pkg/cmd/types.go

```diff
@@ -57,5 +68,39 @@ func (i *ImageInspectorOptions) Validate() error {
 	if len(i.Serve) == 0 && i.Chroot {
 		return fmt.Errorf("Change root can be used only when serving the image through webdav")
 	}
+	if len(i.ScanType) > 0 && len(i.ScanResultsDir) == 0 {
+		return fmt.Errorf("scan-result-dir must be specified with scan-type")
+	}
+	if len(i.ScanResultsDir) > 0 && len(i.ScanType) == 0 {
+		return fmt.Errorf("scan-result-dir can be used only when spacifing scan-type")
+	}
+	if len(i.ScanResultsDir) > 0 {
+		fi, err := os.Stat(i.ScanResultsDir)
+		if err != nil || os.IsNotExist(err) {
```

Comment:
@enoodle please fix that, as we create the directory for the output let's create this as well and use something like:

`sudo ./image-inspector --image=registry.access.redhat.com/openshift3/metrics-cassandra --path=/tmp/image-content --chroot --serve=0.0.0.0:5555 --scan-type=openscap --scan-results-dir=/tmp/image-scan-results`

cc @pweil- I hope you don't have strong feelings about this :smile:  ...mkdir is atomic, so we can do a mkdir (best-effort, it can fail) and then do an `IsDir()` check (to cover both for failed creation, or existing as a file).